config/default/field.field.node.government_gazette.field_files.yml

```diff
@@ -0,0 +1,46 @@
+uuid: c2a5573c-6b56-42ea-8d45-0a93bf4cc3b7
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_files
```

Comment:
I think we'll need to model this as a new field, which is `field_file` (singular) and only allows one value.